src/libjasper/jpc/jpc_dec.c

```diff
@@ -1949,6 +1960,8 @@ static void jpc_dequantize(jas_matrix_t *x, jpc_fix_t absstepsize)
 		for (jas_matind_t j = 0; j < jas_matrix_numcols(x); ++j) {
 			jas_seqent_t t = jas_matrix_get(x, i, j);
 			if (t) {
+				// mid-point reconstruction
+				t = (t > 0) ? jpc_fix_add(t, recparam) : jpc_fix_sub(t, recparam);
 				t = jpc_fix_mul(t, absstepsize);
 			} else {
 				t = 0;
```

Comment:
Redundant code. If not `t` this means it is zero already
```suggestion
```

Reply:
Accepted. Thank you for the suggestion.